ga4gh/backends.py

```diff
@@ -386,8 +397,9 @@ def searchVariants(self, request):
             else:
                 pageToken = int(pageToken)
             request.pageToken = pageToken
-        ds = self._variantSets[request.variantSetIds[variantSetIndex]]
-        response = ds.searchVariants(request)
+        variantSet = self\
```

Comment:
I would use a hanging/visual indent here instead of breaking the line, e.g.
```python
variantSet = self._variantSetMap[
    request.variantSetIds[variantSetIndex]]
```